Exercise_A/index.html

```diff
@@ -3,14 +3,16 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>index</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="" />
   </head>
   <body>
-    <!-- CODE HERE -->
 
-    <script src="" async defer></script>
+    
+    <a href="about.html" target="#blank"><img src="http://www.createordie.com.au/wp-content/uploads/2021/06/landscape-photography-tips-4-1024x620.jpg"></a>
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done!